smartmin/users/models.py

```diff
@@ -25,7 +25,7 @@ class RecoveryToken(models.Model):
 
 
 class FailedLogin(models.Model):
-    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.PROTECT)
+    username = models.CharField(max_length=256, null=True)
```

Comment:
I think to have this as not-null and start with a data migration to delete them all

Reply:
Added